pkg/plugins/runtime/k8s/controllers/inbound_converter.go

```diff
@@ -74,7 +79,7 @@ func inboundForService(zone string, pod *kube_core.Pod, service *kube_core.Servi
 	return
 }
 
-func inboundForServiceless(zone string, pod *kube_core.Pod) (ifaces []*mesh_proto.Dataplane_Networking_Inbound) {
+func inboundForServiceless(zone string, pod *kube_core.Pod, name string) (ifaces []*mesh_proto.Dataplane_Networking_Inbound) {
```

Comment:
IMO this should return a single `Inbound`, not a slice.

And also not use named returns.

Reply:
Fix, thanks for noticing